assign-pull-requests.py

```diff
@@ -314,6 +314,12 @@ def assign_one(pr_getter, issue, dev_mapping, proj_mapping, categories,
                     keywords_add=['PullRequest'],
                     see_also_add=[pr.html_url])
             bz.update_bugs(list(bugs), updq)
+
+        # be careful to match security@, security-audit@, and security-kernel@
+        security = any([bug.assigned_to.startswith('security') and
```

Comment:
I don't think you need the outer `[...]`, `any()` should accept an iterator.

Reply:
You're right. In testing again I found that python-bugzilla was now returning emails in assigned_to without the trailing `@*` part and this would break the endswith match that was here, so instead I changed it to match the Bugzilla user IDs.